console/BUILD

```diff
@@ -83,16 +83,16 @@ distribution(
     output_filename = "grakn-core-console",
 )
 
-linux_packages(
+deb_package(
+    name = "deploy-deb",
```

Comment:
`name = "distribution-deb"`

Reply:
@lolski as discussed on Slack, it's already named correctly